src/adaptors/reference_adaptor.c

```diff
@@ -385,56 +377,52 @@ static void qdr_ref_delivery_update(void *context, qdr_delivery_t *dlv, uint64_t
+                                                      QD_INCOMING,          // qd_direction_t  dir,
+                                                      "",                   // const char     *host,
+                                                      "",                   // const char     *ssl_proto,
+                                                      "",                   // const char     *ssl_cipher,
+                                                      "",                   // const char     *user,
+                                                      "reference-adaptor",  // const char     *container,
+                                                      0,                    // pn_data_t      *connection_properties,
+                                                      0,                    // int             ssl_ssf,
+                                                      false,                // bool            ssl,
+                                                      "",                   // peer router version,
+                                                      false);               // streaming links
+
+    adaptor->conn = qdr_connection_opened(adaptor->core,     // core
+                                          adaptor->adaptor,  // protocol_adaptor
+                                          true,              // incoming
+                                          QDR_ROLE_NORMAL,   // role
+                                          1,                 // cost
                                           qd_server_allocate_connection_id(adaptor->core->qd->server),
-                                          0,                // label
-                                          0,                // remote_container_id
-                                          false,            // strip_annotations_in
-                                          false,            // strip_annotations_out
-                                          250,              // link_capacity
-                                          0,                // policy_spec
-                                          info,             // connection_info
-                                          0,                // context_binder
-                                          0);               // bind_token
+                                          0,      // label
+                                          0,      // remote_container_id
+                                          false,  // strip_annotations_in
+                                          false,  // strip_annotations_out
+                                          250,    // link_capacity
+                                          0,      // policy_spec
+                                          info,   // connection_info
+                                          0,      // context_binder
+                                          0);     // bind_token
```

Comment:
this one parameter per line formatting gets squashed without the per-parameter comments; maybe something to adjust further; at some point the function should be changed to take a struct, instead of these long parameter lists, imo

Reply:
review the following options

```
BinPackArguments: false
BinPackParameters: false
```

```
ExperimentalAutoDetectBinPacking: false
AllowAllParametersOfDeclarationOnNextLine: false
```

```
BreakBeforeBinaryOperators
```

Reply:
There does not seem to be satisfactory solution for this parameter list formatting

* https://stackoverflow.com/questions/24974919/how-to-mark-a-region-so-clang-format-wont-touch-it
* https://stackoverflow.com/questions/72573885/can-clang-format-be-instructed-to-maintain-newlines-in-a-function-call-parameter